litmus-portal/frontend/src/views/Analytics/ApplicationDashboard/ChaosAnnotationsEditor/styles.ts

```diff
@@ -0,0 +1,61 @@
+import { makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles((theme) => ({
+  // drawer
+  drawer: {
+    width: 'fit-content',
+    flexShrink: 0,
+  },
+  drawerPaper: {
+    width: '100%',
+    background: 'rgba(0, 0, 0, 0.6)',
+  },
+  drawerContent: {
+    height: '100%',
+    width: '50%',
+    marginLeft: '50%',
```

Comment:
please use theme.spacing()

Reply:
here the height and width of the screen is unknown so we had to use in percentage of the view port height or width @Saranya-jena  Its same as used for the hub section